src/app/product.model.ts

```diff
@@ -0,0 +1,33 @@
+import {Bid} from './bid.model';
+
+
+export class Product {
+  public id;
+  public name;
+  public description;
+  public startPrice;
+  public auctionStart;
+  public auctionEnd;
+  public pictures: Array<string>;
+  public bids: Array<Bid>;
+
+
+  constructor(id, name, description, startPrice, auctionStart, auctionEnd, pictures: Array<string>, bids: Array<Bid>) {
```

Comment:
As per previous comment, add another constructor here that takes an object as param